README.md

```diff
@@ -1,11 +1,19 @@
 # PORTFOLIO
 
-This is a Webpage about our Porfolio.
+> This is a Webpage about our Porfolio.
 
 ## Built With
 
 - HTML and CSS
 
+## Live Demo
+
+Click [here](https://gedeonts.github.io/PORTFOLIO/)
+
+## Getting Started
+
+You can have a copy of this repo run on your local machine by just directly cloning it.
+
```

Comment:
- [OPTIONAL] Kindly, Add a descriptive README file to your project, it will be great to add instructions to set the project up locally. This includes `git clone .......` , `cd .....` etc